### PR TITLE
parsigs universe

### PR DESCRIPTION
**Pull request by royashcenazi**
## Description
Add Parsigs project to the Universe section.

### Types of change
Enhancement

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by bdura** [comment]
Hi @royashcenazi, don't worry about the double PR.

You can tell Hatch to use direct link:

```toml
[build-system]
requires = ["hatchling"]
build-backend = "hatchling.build"

[project]
# ...
dependencies = [
    "spacy~=3.0",  # restrict the version to spaCy >=3.0,<4.0 to avoid bad surprises
    "word2number",
    "en_parsigs@https://huggingface.co/royashcenazi/en_parsigs/resolve/main/en_parsigs-any-py3-none-any.whl"
]
# ...

[tool.hatch.metadata]
# This is required
allow-direct-references = true
```

Hope this helps!

**Review by bdura — COMMENTED** [review]
We need just need to resolve the conflicts and make sure your model is installed by default for the example to be working, and we're good to go!

**Comment by royashcenazi** [comment]
> 

@bdura Done :)
Lmk if the explanation works well or need some rephrasing

File: website/meta/universe.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
             "id": "parsigs",
             "title": "parsigs",
             "slogan": "Structuring prescriptions text made simple using spaCy",
-            "description": "Parsigs is an open-source project that aims to extract the relevant dosage information from prescriptions text without compromising the patient's privacy.",
+            "description": "Parsigs is an open-source project that aims to extract the relevant dosage information from prescriptions text without compromising the patient's privacy. Notice you also need to install the model in order to use tha package: `pip install https://huggingface.co/royashcenazi/en_parsigs/resolve/main/en_parsigs-any-py3-none-any.whl`",
```
Comment (bdura) [inline website/meta/universe.json:7]: There is a small typo:

```suggestion
            "description": "Parsigs is an open-source project that aims to extract the relevant dosage information from prescriptions text without compromising the patient's privacy.\n\nNotice you also need to install the model in order to use the package: `pip install https://huggingface.co/royashcenazi/en_parsigs/resolve/main/en_parsigs-any-py3-none-any.whl`",
```

**Review by bdura — CHANGES_REQUESTED** [review]
There is a small typo. You might want to repeat the warning within the example itself to help users who just copy-paste without reading the description:

```python
from parsigs.parse_sig_api import StructuredSig, SigParser

# You'll need to install the trained model, see instructions
sig_parser = SigParser()

# ...
```

We can merge your submission once the typo is resolved!

**Comment by royashcenazi** [comment]
> There is a small typo. You might want to repeat the warning within the example itself to help users who just copy-paste without reading the description:
> 
> ```python
> from parsigs.parse_sig_api import StructuredSig, SigParser
> 
> # You'll need to install the trained model, see instructions
> sig_parser = SigParser()
> 
> # ...
> ```
> 
> We can merge your submission once the typo is resolved and conflicts are handled! You can merge master into your branch to do that, and push the merge commit.

@bdura committed your suggestion, added a reference to the installation command in the code example and merged master, thanks

**Pull request merged**